nrpe/packaging/centreon-nrpe4-daemon.yaml

```diff
@@ -0,0 +1,92 @@
+name: "centreon-nrpe4-daemon"
+arch: "amd64"
+platform: "linux"
+version_schema: "none"
+version: "4.1.0"
+release: "${RELEASE}${DIST}"
+section: "default"
+priority: "optional"
+maintainer: "Centreon <contact@centreon.com>"
+description: |
+  The centreon-nrpe packages contains the Nagios Remote Plug-ins Executor.
+  Daemon which can execute predefined commands on the remote host.
+  Commit: @COMMIT_HASH@
+vendor: "Centreon"
+homepage: "https://centreon.com"
+license: "GPLv2+"
```

Comment:
Couldn't we use Apache v2 instead ?

Reply:
Can we if the source license is different? https://github.com/NagiosEnterprises/nrpe?tab=GPL-2.0-1-ov-file